app/celery/tasks.py

```diff
@@ -46,6 +47,10 @@
 DEFAULT_SHATTER_JOB_ROWS_BATCH_SIZE = 8
 
 
+class JobRowTooBig(Exception):
```

Comment:
I’d consider renaming this to something more generic:
```suggestion
class CantProcessJobRow(Exception):
```

We are sniffing `InvalidParameterValue` which is quite generic. If we messed up something else we’d end up with an exception like:
```
JobRowTooBig: The specified queue does not exist or you do not have access to it.
```

This might make send whoever is on support down a bit of a rabbit hole trying to figure out which row is too big (when it’s nothing to do with the row size).